### DISCUSSION TITLE
播放4K视频卡顿

### DISCUSSION THREAD

**Discussion by Tpfdd** [ywmoyue/biliuwp-lite#323] [General]
播放4K视频卡顿，一直缓冲没反应，有时候只加载音频，其它视频正常，切换为1080P正常。

**Comment by ywmoyue** [comment]
@Tpfdd 发一下日志文件看看

**Comment by Tpfdd** [comment]
[20230918.log](https://github.com/ywmoyue/biliuwp-lite/files/12644618/20230918.log)


> @Tpfdd 发一下日志文件看看

**Comment by Tpfdd** [comment]
还有就是关注的UP主为什么在关注列表找不到，例如@俞俐均官方帳號
关注列表能不能默认最常访问
目前不能播放HDR视频吗

**Comment by ywmoyue** [comment]
这个up找不到可能是这个问题：https://github.com/ywmoyue/biliuwp-lite/discussions/322

**Comment by ywmoyue** [comment]
HDR视频没研究过，麻烦给个可以播放HDR视频的av/bv号我测测

**Reply by Tpfdd** [reply]
4K HDR BV15z4y1Z734
云之幻的可以开启，可以研究一下

**Reply by ywmoyue** [reply]
@Tpfdd 我试了下hdr选项是有的，需要将编码设为h265 ，我现在身边没有hdr显示器只能确认hdr选项是有的，没法确认是否有效
